style.css

```diff
@@ -0,0 +1,93 @@
+body{
+  display: flex;
+  flex-direction: column;
+  margin:0;
+  font-family: Arial, Helvetica, sans-serif ;
+}
+.container-home {
+  display: flex;
+  flex-direction: column;
+  margin:0;
+  background-image: url(/images/carousel_img_1.png);
+  background-repeat: no-repeat;
+  height: 600px;
+  background-size: contain;
+  padding-left:30px;
+  
+}
+.container-home h4 {
```

Comment:
It's possible to style this way, but mostly it's done by giving class names to these elements and then style it using them. Just a suggestion to think about :)